server.ts

```diff
@@ -179,21 +179,23 @@ app.post("/conferences", async (req, res) => {
       let conferenceId = slugsToIds.get(slug)!;
       try {
         await getConference(conferenceId);
-        res.status(409).send();
-        return;
+        return res.status(200).send({
+          id: conferenceId,
+          slug: slug,
+        });
       } catch (e) {
         slugsToIds.delete(slug);
       }
     }
 
     let conferenceId = await createConference(slug);
-    res.status(200).send({
+    return res.status(200).send({
```

Comment:
I don't think it matters but is the return of the `res.status().send()` required?

Reply:
the return isn't but I usually like to just put both in-line for express apps just as a coding style

Reply:
Not having to return the response is actually one of my favorite things about express, as it allows you to continue processing afterward, which would be fantastic in many places in webrtc manager for example. In there particular instances it makes no difference, but I wouldn't necessarily get in the habit of it either. My 2¢, although I've never seen that anywhere else either.